examples/laplaceMPI/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.1.3)
 
 if (KOKKOS)
-  find_package(Kokkos REQUIRED) #new
```

Comment:
Is kokkos no longer required for the examples, or is this flag handled elsewhere?

Reply:
looks like I need to move that subdirectory command into the big kokkos if block in the parent examples CMakeList.